django/contrib/auth/views.py

```diff
@@ -128,25 +126,15 @@ class LogoutView(RedirectURLMixin, TemplateView):
     Log out the user and display the 'You are logged out' message.
     """
 
-    # RemovedInDjango50Warning: when the deprecation ends, remove "get" and
-    # "head" from http_method_names.
-    http_method_names = ["get", "head", "post", "options"]
+    http_method_names = ["post", "options"]
     template_name = "registration/logged_out.html"
     extra_context = None
 
-    # RemovedInDjango50Warning: when the deprecation ends, move
-    # @method_decorator(csrf_protect) from post() to dispatch().
+    @method_decorator(csrf_protect)
```

Comment:
There was some at length discussion on the PR before this (that I'd need to review to recall perfectly) but I was pretty sure we'd agreed this was sound. 

In any case, it should be fine, since CSRF middleware has: 

```
        if request.method in ("GET", "HEAD", "OPTIONS", "TRACE"):
            return self._accept(request)
```